.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report incorrect behavior in the pandas library
+title: "BUG: "
+labels: [Bug, Needs Triage]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the
+            [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the master branch of pandas.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a copy-pastable example that can be run on a Python REPL.
+      placeholder: >
+        import pandas as pd
+
+        df = pd.DataFrame(range(5))
+
+        ...
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Issue Description
+      description: >
+        Please provide a description of the issue with the reproducible example.
```

Comment:
Might be just me, but the wording "with the reproducible example" makes it almost sound like they should include both a description and the example itself again here. Maybe `Please provide a description of the issue shown in the reproducible example`?